assemblies/features/src/main/resources/features.xml

```diff
@@ -61,6 +61,8 @@
     <bundle>mvn:pentaho/pentaho-big-data-kettle-plugins-common-job/${project.version}</bundle>
     <bundle>mvn:pentaho/pentaho-big-data-kettle-plugins-common-named-cluster-bridge/${project.version}</bundle>
     <bundle>mvn:pentaho/pentaho-big-data-kettle-plugins-guiTestActionHandlers/${project.version}</bundle>
+    <bundle>wrap:mvn:pentaho/pentaho-hadoop-shims-api/${project.version}</bundle>
```

Comment:
api will be now loaded from both big-data-plugin lib folder and from osgi, right? we have legacy-plugin jar which use pentaho-hadoop-shims-api loaded not from osgi, and calls to api library classes from it, according dependency in maven https://github.com/pentaho/big-data-plugin/blob/master/legacy/pom.xml#L51, while other bundles will use api classes loaded from another classloader - osgi, which can come to classcastexceptions, full tests of impersonated shims and ordinary shim are highly required